app/src/layouts/common/projectSelector/projectSelector.jsx

```diff
@@ -23,10 +23,57 @@ import { PROJECT_PAGE } from 'controllers/pages/constants';
 import { SIDEBAR_EVENTS } from 'components/main/analytics/events';
 import { ScrollWrapper } from 'components/main/scrollWrapper';
 import { NavLink } from 'components/main/navLink';
+import { withTooltip } from 'components/main/tooltips/tooltip';
 import styles from './projectSelector.scss';
 
 const cx = classNames.bind(styles);
 
+const Tooltip = ({ activeProject }) => (
+  <div className={cx('project-selector-tooltip')}>{activeProject}</div>
+);
+Tooltip.propTypes = {
+  activeProject: PropTypes.string.isRequired,
+};
+
+const CurrentProjectBlock = ({ getProjectName, toggleShowList, setControlNode, refHandler }) => {
+  return (
+    <div
+      ref={(node) => {
```

Comment:
Is it obligatory to pass all these props associated with the project list (`toggleShowList, setControlNode, refHandler`) to this component?
I think this `node` and related things have responsibilities in the `projectSelector` component and should be left directly in it.